website/frontend/src/pages/OurSolutions/CommunityPage.js

```diff
@@ -26,7 +26,7 @@ const CommunityPage = () => {
             <div className="CommunityPage">
                 <SEO
                     title="Our Solutions"
-                    siteTitle="African Cities"
+                    siteTitle="For Communities"
```

Comment:
What inspired this content change? Figma designs?

Reply:
This is for SEO. The Communities page had been given the title 'African Cities' which was an error @Baalmart